packages/flutter/test/widgets/semantics_test.dart

```diff
@@ -594,7 +594,6 @@ void main() {
         ),
       ],
     );
-    expect(semantics, hasSemantics(expectedSemantics, ignoreId: true));
```

Comment:
Why is this check being removed?

Reply:
It isn't. I was about to push an update.